pkg/controller/ec2/routetable/controller.go

```diff
@@ -94,7 +96,7 @@ type external struct {
 }
 
 func (e *external) Observe(ctx context.Context, mgd resource.Managed) (managed.ExternalObservation, error) { // nolint:gocyclo
-	cr, ok := mgd.(*v1alpha4.RouteTable)
+	cr, ok := mgd.(*v1alpha5.RouteTable)
 	if !ok {
 		return managed.ExternalObservation{}, errors.New(errUnexpectedObject)
 	}
```

Comment:
Unrelated to this PR but the err string used in the result of `ec2.IsRtUpToDate(cr.Spec.ForProvider, observed)` is `errDescribe`, which is misleading.

Reply:
went ahead and added better message while i'm at it